set.mm

```diff
@@ -30533,22 +30537,24 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
     $( Equality theorem for restricted class abstractions.  Inference form of
        ~ rabeqf .  (Contributed by Glauco Siliprandi, 26-Jun-2021.) $)
     rabeqif $p |- { x e. A | ph } = { x e. B | ph } $=
-      ( wceq crab rabeqf ax-mp ) CDHABCIABDIHGABCDEFJK $.
+      ( wceq crab cv wcel wa cab nfth eleq2 anbi1d abbid df-rab 3eqtr4g ax-mp )
```

Comment:
It seems that hypotheses rabeqf.1 and rabeqf.2 ar not referenced in  the proof.  Get rid of them.

Reply:
I don't understand, if I write:

<pre>MM> sh pr rabeqif /lem
13 rabeqif.3     $e |- A = B
18 rabeqf.1      $e |- F/_ x A
19 rabeqf.2      $e |- F/_ x B
20 18,19 rabeqf  $p |- ( A = B -> { x e. A | ph } = { x e. B | ph } )
21 13,20 ax-mp   $a |- { x e. A | ph } = { x e. B | ph }</pre>

They do appear in the proof, also when I hide them from `rabeqif` the metamath-exe verifier complains.

Reply:
The reason why they appear in the old proof, is to show F/ x A = B .  But since A = B is a hypothesis, your approach via nfth effectively arrives at the same result, without the hypotheses in question.  They can be dropped from the new (but not the old) proof.

Reply:
You're right, I applied the suggestions in commits c2597c9 and 5cc41dd.

Indeed revising the proofs of `extwwlkfab` `extwwlkfabOLD` `smfliminf` wasn't easy at all, but I managed to do it in the end.